backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -23,10 +27,15 @@ class CardMutationService {
             if (!Authorizer.mayCreateCardInRegion(user, targetedRegionId)) {
                 throw UnauthorizedException()
             }
-            val totpSecret = if (card.totpSecretBase64 != null) Base64.getDecoder().decode(card.totpSecretBase64) else null
+            val activationSecret =
+                if (card.activationSecretBase64 != null)
+                    Base64.getDecoder().decode(card.activationSecretBase64)
+                else null
```

Comment:
A bit more compact, but maybe little bit less readable:
```suggestion
            val activationSecret = card.activationSecretBase64?.let { Base64.getDecoder().decode(it) }
```